tests/data/3.extracted_features/gait/arm_activity_meta.json

```diff
@@ -28,10 +28,10 @@
             "channels": [
                 "angle_perc_power",
                 "range_of_motion",
-                "forward_peak_ang_vel_mean",
-                "forward_peak_ang_vel_std",
-                "backward_peak_ang_vel_mean",
-                "backward_peak_ang_vel_std",
+                "forward_peak_velocity_mean",
```

Comment:
Uit interesse: waarom wil je het hernoemen naar velocity i.p.v. angular velocity? Kan dit nog voor verwarring zorgen qua type sensor data wat je gebruikt (gyroscoop/accelerometer)?

Reply:
De reden waarom ik het _velocity_ wil noemen is om twee redenen: (1) eenvoudiger te begrijpen (ang_vel kan van alles betekenen), en (2) minder underscores (_). Er zal geen verwarring zijn, omdat het in plaats van angular velocity nu om velocity gaat, en dit heeft neits met accelerometer te maken. Deze berekeningen zijn puur voor de gyroscoop.

Reply:
Duidelijk ;)